web/src/list_widget.ts

```diff
@@ -184,14 +199,15 @@ export function create<Key = unknown, Item = Key>(
         old_widget.clear_event_handlers();
     }
 
-    const meta = {
+    const meta: ListWidgetMeta<Key, Item> = {
         sorting_function: null,
         sorting_functions: new Map(),
         offset: 0,
         list,
         filtered_list: [],
         reverse_mode: false,
         filter_value: "",
+        $scroll_container: scroll_util.get_scroll_element(opts.$simplebar_container),
```

Comment:
Where does this line come from?

Reply:
This was coming from `set_up_event_handlers` function well I actually extracted this change in a separate commit now with a clear message describing this change.